<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HiringDialog</class>
 <widget class="QDialog" name="HiringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Прийняття на роботу</string>
  </property>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>511</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="adress_label">
        <property name="text">
         <string>Адреса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="adress_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="passport_label">
        <property name="text">
         <string>Паспорт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passport_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="education_label">
        <property name="text">
         <string>Освіта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="education_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="medical_label">
        <property name="text">
         <string>Медична страховка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="medical_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="department_label">
        <property name="text">
         <string>Департамент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="department_cb"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="position_label">
        <property name="text">
         <string>Позиція</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="position_cb"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="salary_label">
        <property name="text">
         <string>Заробітня платня</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>510</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Найняти</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="name_label_2">
        <property name="text">
         <string>Ім'я</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_edit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="surname_label_2">
        <property name="text">
         <string>Призвіще</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="surname_edit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="patronymic_label_2">
        <property name="text">
         <string>По-батькові</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="patronymic_edit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="birth_label_2">
        <property name="text">
         <string>Дата народження</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="date_edit_2">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1990</year>
          <month>6</month>
          <day>15</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>1997</year>
          <month>12</month>
          <day>18</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>1900</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>dd.MM.yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="phone_label_2">
        <property name="text">
         <string>Номер телефону</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phone_edit_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QRadioButton" name="radioButton_new">
       <property name="text">
        <string>Введіть нового:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_existed">
       <property name="text">
        <string>Або виберіть з існуючих:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>tableView_2</zorder>
  <zorder>line</zorder>
  <zorder>pushButton</zorder>
  <zorder>error_label</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
